sonarqube-companion-rest/src/main/java/pl/consdata/ico/sqcompanion/sync/SyncAfterAppInitialization.java

```diff
@@ -20,6 +21,9 @@ public SyncAfterAppInitialization(final SynchronizationService synchronizationSe
 		this.synchronizationService = synchronizationService;
 	}
 
+	/**
+	 * TODO (mp): to nie powinno blokować wstawania aplikacji
```

Comment:
Tu ciężko powiedzieć, bo start robisz co jakiś czas a jak pozwolisz wstać bez żadnego cache to będziesz miał pustą apkę ;-) Pewnie wrócimy jak pomyślimy o cache ostatnich wyników, żeby szybciej wstawało ;-)